src/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+// Copyright 2022 Google LLC
```

Comment:
Shouldn't we write `2019-2022`?

Reply:
Ah I thought this file is from this year, but we actually had it for longer. Fixed here and in other touched files, and I also fixed some documentation that I noticed.